tensorpack/dataflow/common.py

```diff
@@ -471,3 +471,49 @@ def add_next():
                     for v in self.q:
                         yield v
                     return
+
+
+class DebugData(ProxyDataFlow):
```

Comment:
Could you name it something more informative about what is happening? Such as PrintData/LogData/PrintDataShape/PrintDataInfo, similar to TestDataSpeed I'm having.
PrintData might not be very bad because in the future, options can be added to print k datapoints instead of just 1.